controller/test/pid_lib/pid_test.cpp

```diff
@@ -181,13 +181,107 @@ TEST(PidTest, DerivativeOnError) {
                     sample_period.seconds());
 }
 
+TEST(PidTest, CallFasterThanSample) {
```

Comment:
Are these just the same old tests, or is there anything new?

Reply:
I split it into two commits, a clean revert and my changes.